frontend/Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM node:10-alpine
+FROM node:latest
 
 WORKDIR /usr/src/app
 
 COPY package*.json ./
 COPY . .
 
-RUN npm install
+RUN npm install -g npm@8.10.0 && npm install
```

Comment:
rethinking it
remove the npm@8.10.0
it is not mandatory and we are taking node latest